app/backend/src/clj/teet/contract/contract_commands.clj

```diff
@@ -174,6 +174,16 @@
               {:db/id company-contract-eid
                :company-contract/employees "new-company-contract-employee"}]})
 
+(defcommand :thk.contract/change-person-status
+  {:doc "Activate/Deactivate contract person"
+   :payload {user-id :user-id
+             active? :active?}
+   :context {:keys [user db]}
+   :project-id nil
+   :authorization {:contracts/contract-editing {}}
+   :transact [{:db/id user-id
```

Comment:
it should be :db/id of company-contract-employee entity defined here: https://github.com/solita/mnt-teet/blob/master/app/backend/resources/schema.edn#L2020